HW_7/task7.1.py

```diff
@@ -0,0 +1,13 @@
+def check_the_phrase(phrase):
+    return len(list(filter(lambda letter: letter in ["а", "у", "и", "ы", "о", "э", "ю", "я", "е", "ё"], phrase)))
```

Comment:
Давайте сделаем чуть короче.
```suggestion
    return len(list(filter(lambda letter: letter in "аяуюоеёэиы", phrase)))
```